doc/API.md

```diff
@@ -404,14 +405,15 @@ You'll have to manually fetch the vaults statuses if you want to know, for examp
 
 ##### Spend transaction resources
 
-| Field               | Type          | Description                                                          |
-| ------------------- | ------------- | -------------------------------------------------------------------- |
-| `deposit_outpoints` | string array  | Array of the deposit outpoints of the vaults this transaction spends |
-| `deposit_amount`    | integer       | Total amount in satoshis of the vaults this transaction spends      |
-| `psbt`              | string        | Base64-encoded Spend transaction PSBT                                |
-| `change_index`      | integer       | Index of the change output, might be null                            |
-| `cpfp_index`        | integer       | Index of the CPFP outputs                                            |
-| `status`            | string        | [Spend status](#spend_status)                                        |
+| Field               | Type         | Description                                                                                             |
+| ------------------- | ------------ | ------------------------------------------------------------------------------------------------------- |
+| `deposit_outpoints` | string array | Array of the deposit outpoints of the vaults this transaction spends                                    |
+| `deposit_amount`    | integer      | Total amount in satoshis of the vaults this transaction spends                                          |
+| `psbt`              | string       | Base64-encoded Spend transaction PSBT                                                                   |
+| `change_index`      | integer      | Index of the change output, might be null                                                               |
+| `cpfp_index`        | integer      | Index of the CPFP outputs                                                                               |
+| `status`            | string       | [Spend status](#spend_status)                                                                           |
+| `deprecated`        | bool         | If true the Spend transaction cannot be used anymore, one of its input was spent by another transaction |
```

Comment:
Why is it another boolean instead of another status?

Reply:
I read the discussion, do you prefer `orphaned` ?

Reply:
No it's good like that no worries. The french for deprecated sounds good to me too :) https://www.linguee.fr/francais-anglais/search?source=auto&query=deprecated